tests/test_simulation.py

```diff
@@ -416,11 +422,458 @@ def test_element_nodal_forces_elemental(self, allkindofcomplexity):
         static_simulation = dpf.load_simulation(data_sources=allkindofcomplexity)
         element_nodal_forces = static_simulation.element_nodal_forces_elemental()
         assert len(element_nodal_forces._fc) == 3
-        assert element_nodal_forces._fc.time_freq_support.time_frequencies.data == 1
+        assert element_nodal_forces._fc.get_time_scoping().ids == [1]
         field = element_nodal_forces._fc[0]
         op = static_simulation._model.operator("ENF")
         op.connect(9, core.locations.elemental)
         field_ref = op.eval()[0]
         assert field.component_count == 3
         assert field.data.shape == (9433, 3)
         assert np.allclose(field.data, field_ref.data)
+
+
+class TestTransientMechanicalSimulation:
+    def test_displacement(self, transient_simulation):
+        result = transient_simulation.displacement(
+            components=["X"],
+            node_ids=[2, 3, 4],
+            all_sets=True,
```

Comment:
@cbellot000 is that what you had in mind? I added a boolean to ask explicitly for every set. This way it will be generic to all types of simulations, and it is very explicit.